editor/src/clj/editor/debug_view.clj

```diff
@@ -47,7 +47,7 @@
 (def ^:private break-label "Break")
 (def ^:private continue-label "Continue")
 (def ^:private detach-debugger-label "Detach Debugger")
-(def ^:private start-debugger-label "Start / Attach")
+(def ^:private start-debugger-label "Start/Attach")
```

Comment:
Unrelated: Fixed inconsistent spacing around slash. I checked a bunch of other applications, and apparently there shouldn't be spaces around the slash when used like this.